scripts/globals/mobskills/nepenthean_hum.lua

```diff
@@ -0,0 +1,26 @@
+---------------------------------------------
+-- Nepenthean Hum
+-- Description: Inflicts amnesia 10' AOE
+-- Type: Enfeebling
+-- Utsusemi/Blink absorb: Ignores shadows
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    if VanadielHour() >= 6 and VanadielHour() <= 18 then
+        return 0
+    else
+        return 1
+    end
+end
+
+function onMobWeaponSkill(target, mob, skill)
+    local message = dsp.msg.basic.SKILL_MISS
```

Comment:
This isn't needed, you are already setting the message to whatever MobStatusEffectMove() returns below